source/includes/fact-migrate-atlas.rst

```diff
@@ -0,0 +1,5 @@
+You can use |tool-command| to migrate from a self-hosted deployment
+to `MongoDB Atlas <https://www.mongodb.com/docs/atlas>`__. MongoDB
+Atlas is the fully managed service for MongoDB deployments in the
+cloud. To learn more, see `Seed with mongorestore 
+<https://www.mongodb.com/docs/atlas/import/mongorestore/>`__.
```

Comment:
Should we also add a link to the documentation for Atlas-managed Migration services? https://www.mongodb.com/cloud/atlas/migrate or mongodb.com/docs/atlas/import/

It looks like the link here specifically directs users to using mongorestore to seed their db, but doesn't refer to C2C or managed Atlas migration at all